spec/views/users/users_index_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe 'users#index', type: :feature do
+  describe 'User Test' do
+    before(:each) do
+      @user1 = User.create(name: 'Sammy', photo: 'sammy.png', bio: 'Sammy bio', posts_counter: 0,
+                           email: 'sammy@gmail.com', password: 'test12345')
+      @user2 = User.create(name: 'Ahmed', bio: 'Ahmed bio', photo: 'sammy.png', email: 'ahmed@gmail.com',
+                           password: 'test12345')
```

Comment:
- [ ] currently, when I run these users-index tests on this page all of them fail. please make sure that these tests are working to ensure your application runs as expected.